test/checks/navigation/internal-link-present.js

```diff
@@ -18,6 +18,16 @@ describe('internal-link-present', function () {
 		assert.isTrue(checks['internal-link-present'].evaluate.apply(checkContext, params));
 	});
 
+	it('should return false when a hashbang URL was used', function () {
+		var params = checkSetup('<div id="target"><a href="#!foo">hi</a></div>');
+		assert.isFalse(checks['internal-link-present'].evaluate.apply(checkContext, params));
+	});
+
+	it('should return false when a hash route URL was used', function () {
+		var params = checkSetup('<div id="target"><a href="#/home">hi</a></div>');
```

Comment:
I'd like to see one more test case to mimic [OP's issue](https://twitter.com/ackleyk/status/981549994711896064): `href="#!/contact-us"`